en/docs/administer/managing-users-and-roles/managing-permissions.md

```diff
@@ -63,7 +63,7 @@ You will be able to log in to the management console of the product with the **a
 
 The **Login** permission defined under **Admin** permissions allows users to log in to the management console of the product. Therefore, this is the primary permission required for using the management console.
 
-![Login Permissions](../../../assets/img/administer/login-permissions.png)
```

Comment:
Was the removal of ../ intentional here?

Reply:
yes.. now it's under docs->administer->managing-users-and-roles

Reply:
@shaniR , but wouldn't it have been better to change ../../../ to {{base_path}} instead of ../../?

Reply:
yes.. that needs to be fixed separately. This PR focuses on the restructuring. Didn't check the content.

Reply:
Noted